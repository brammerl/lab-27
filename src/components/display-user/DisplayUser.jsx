import React from 'react';

const DisplayUser = ({userInfo}) => {
  return (
  <>
    <h1>{userInfo.name}</h1>
    <ul>
      <li>Followers: {userInfo.followers}</li>
      <li>Following: {userInfo.following}</li>
      <li><a href={userInfo.html_url}>Site</a></li>
    </ul>
  </>
  )

}

export default DisplayUser;