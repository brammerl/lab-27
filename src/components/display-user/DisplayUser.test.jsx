import React from 'react';
import { shallow } from 'enzyme';
import DisplayUser from './DisplayUser';

describe('DisplayUser Component', () => {
  it('renders the display user component', () => {
    const wrapper = shallow(<DisplayUser userInfo={{}}/>)
    expect(wrapper).toMatchSnapshot();
  })
})