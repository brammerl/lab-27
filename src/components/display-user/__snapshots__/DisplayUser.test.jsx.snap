// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayUser Component renders the display user component 1`] = `
<Fragment>
  <h1 />
  <ul>
    <li>
      Followers: 
    </li>
    <li>
      Following: 
    </li>
    <li>
      <a>
        Site
      </a>
    </li>
  </ul>
</Fragment>
`;
